src/node_task_runner.cc

```diff
@@ -206,7 +206,35 @@ void ProcessRunner::OnExit(int64_t exit_status, int term_signal) {
 
 void ProcessRunner::Run() {
   // keeps the string alive until destructor
-  cwd = package_json_path_.parent_path().string();
```

Comment:
This will not work since this doesn't add the current node_modules to path environment variable.